tests/baselines/reference/inferringClassMembersFromAssignments.js

```diff
@@ -124,6 +124,7 @@ var stringOrNumberOrUndefined = C.inStaticNestedArrowFunction;
 
 
 //// [output.js]
+var _this = this;

```

Comment:
which tests were incorrect before this change? This is the closest one that I've found.